app/models/reports/duplicate_claims.rb

```diff
@@ -0,0 +1,47 @@
+require "csv"
+require "excel_utils"
+
+module Reports
+  class DuplicateClaims
+    include Admin::ClaimsHelper
+
+    NAME = "Duplicate Approved Claims"
+    HEADERS = [
+      "Claim reference",
+      "Teacher reference number",
+      "Full name",
+      "Policy name",
+      "Claim amount",
+      "Claim status",
+      "Decision date",
+      "Decision agent"
+    ].freeze
+
+    def initialize
+      @claims = Claim.approved.select { |claim| Claim::MatchingAttributeFinder.new(claim).matching_claims.any? }
```

Comment:
```suggestion
      @claims = Claim.approved.includes(:eligibility).select { |claim| Claim::MatchingAttributeFinder.new(claim).matching_claims.any? }
```

Not sure if we want to include `decisions` too or if it's less work to filter first and make the additional trips to the db